main.go

```diff
@@ -69,15 +74,11 @@ func (mi *ModuleInstance) Exports() k6modules.Exports {
 
 // Launch Chromium with given options.
 func (m *JSModule) Launch(opts goja.Value) api.Browser {
-	/*go func() {
-		f, err := os.Create("./cpu.profile")
-		if err != nil {
-			log.Fatal(err)
-		}
-		pprof.StartCPUProfile(f)
-		defer pprof.StopCPUProfile()
-		<-ctx.Done()
-	}()*/
+	if _, ok := os.LookupEnv("XK6_BROWSER_PPROF"); ok {
+		go func() {
+			log.Println(http.ListenAndServe("localhost:6060", nil))
```

Comment:
This can't be in `Launch()`, otherwise a new server will try to run on each new VU/iteration.

Maybe make it part of `NewModuleInstance` or `init()`?

Also let's print some debug info to let us know the server was actually started.

Reply:
I've moved it to a new `init` function in main,go. I've just used `log.Println` to log when the pprof server starts -- the actual logger gets created when the browser processes is started. Let me know if I've missed something though.